build/distributions/CentOS/6/x86_64/DVD/isolinux/isolinux.cfg

```diff
@@ -8,16 +8,16 @@ F4 param.msg
 F5 rescue.msg
 label simp
   kernel vmlinuz
-  append ks=cdrom:/ks/dvd/auto.cfg initrd=initrd.img audit=1
+  append ks=cdrom:/ks/dvd/auto.cfg initrd=initrd.img audit=1 simp_install=iso
```

Comment:
Much more reliable approach that is less likely to conflict with ISOs built by others.